airflow/exceptions.py

```diff
@@ -458,3 +458,7 @@ def __str__(self):
             "Error deserializing result. Note that result deserialization "
             "is not supported across major Python versions. Cause: " + str(self.__cause__)
         )
+
+
+class UnknownExecutorException(ValueError):
+    """Raised when an executor is attempted to be loaded but that executor is not configured."""
```

Comment:
Nit:  I hate complaining without offering an alternative, but "attempted to be loaded" feels awkward.

Reply:
Yeah, it's tricky to word it right without an actor in the sentence. Definitely open to suggestions!

What about:
```suggestion
    """Raised when an attempt is made to load an executor that is not configured."""
```

Reply:
Yeah, I get that.  Slight tweak on your idea:   Raised when an attempt is made to load an executor which is not configured.  Or maybe a completely different route and use some variant of  "An unconfigured executor can not be loaded." perhaps?

Either way, don't let this block merging, feel free to just resolve it.

Reply:
Updated the wording!